dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -326,17 +356,26 @@ class ResourceGroup final : private boost::noncopyable
         return bucket_mode == trickle_mode && tp < stop_trickle_timepoint;
     }
 
-    void collectMetrics() const
+    bool needNotifyStopTrickleMode(const std::chrono::steady_clock::time_point & tp)
```

Comment:
```suggestion
    bool overTrickleModeLease(const std::chrono::steady_clock::time_point & tp)
```
just like `inTrickleModeLease`?

Reply:
changed